src/main/java/seedu/duke/userinterface/InputParser.java

```diff
@@ -4,14 +4,7 @@
 import seedu.duke.exceptions.InvalidCommandException;
 import seedu.duke.exceptions.TaskTitleException;
 import seedu.duke.exceptions.TaskWrongFormatException;
-import seedu.duke.userinterface.command.Add;
-import seedu.duke.userinterface.command.CliCommand;
-import seedu.duke.userinterface.command.Exit;
-import seedu.duke.userinterface.command.Help;
-import seedu.duke.userinterface.command.List;
-import seedu.duke.userinterface.command.ListTimetable;
-import seedu.duke.userinterface.command.Remove;
-import seedu.duke.userinterface.command.RemoveTask;
+import seedu.duke.userinterface.command.*;
```

Comment:
I think imports should be listed explicitly, as stated in https://se-education.org/guides/conventions/java/basic.html